Cargo.toml

```diff
@@ -15,3 +18,10 @@ uart_16550 = "0.2.10"
 version = "0.1.2"
 default-features = false
 features = ["regular", "size_14"]
+
+
+[profile.dev]
+panic = "abort"
+
+[profile.release]
+panic = "abort"
```

Comment:
Why are those needed?

Reply:
At first I run `cargo build` will get some error like language item required, but not found: `eh_personality`, adding these lines fixed this error. But now I can't reproduct this error, seems not necessary, will remove later.